ros2topic/ros2topic/verb/echo.py

```diff
@@ -82,6 +85,61 @@ def add_arguments(self, parser, cli_name):
         parser.add_argument(
             '--raw', action='store_true', help='Echo the raw binary representation')
 
+    def choose_qos(self, node, args):
+
+        if (args.qos_profile or args.qos_reliability or args.qos_durability or
+                args.qos_depth or args.qos_history):
+            return qos_profile_from_short_keys(args.qos_profile,
+                                               reliability=args.qos_reliability,
+                                               durability=args.qos_durability,
+                                               depth=args.qos_depth,
+                                               history=args.qos_history)
+
+        qos_profile = QoSPresetProfiles.get_from_short_key('sensor_data')
+        reliability_reliable_endpoints_count = 0
+        durability_transient_local_endpoints_count = 0
+
+        pubs_info = node.get_publishers_info_by_topic(args.topic_name)
+        publishers_count = len(pubs_info)
+        if publishers_count == 0:
+            return qos_profile
+
+        for info in pubs_info:
+            if (info.qos_profile.reliability ==
+                    QoSReliabilityPolicy.RELIABLE):
```

Comment:
nit: I think this will fit on one line, staying under our 100 character limit.

Reply:
Addressed on [160fce7](https://github.com/ros2/ros2cli/pull/613/commits/160fce7d4411ce53251d12ce92dd55b8e77fc046)